test/utils.test.ts

```diff
@@ -44,8 +44,10 @@ export function breaksOnlyOneRule(
   rule: string
 ): void {
   assert.equal(result.conforms, false, result.toString());
-  assert.equal(result.results.length, 1, result.toString());
-  assert.equal(result.results[0].validationId, rule, result.toString());
+  if (!result.conforms) {
```

Comment:
Why do we need to check for conforms? is it throwing any error with the new versioin
Do we have the following tests 

1. undefined result.confirms 
2. null result.conforms

Reply:
doesn't seem to be failing without the check. So I will revert the change.